backend/src/main/java/ch/zuehlke/fullstack/ConnectZuehlke/domain/Employee.java

```diff
@@ -4,13 +4,21 @@
 
     private String firstName;
     private String lastName;
+    private int id;
+    private String code;
 
     private Employee() {
     }
 
-    public Employee(String firstName, String lastName) {
+    public Employee(String firstName, String lastName, int id) {
+        this(firstName, lastName, id, firstName.substring(0, 1) + lastName.substring(0, 2));
+    }
+
+    public Employee(String firstName, String lastName, int id, String code) {
         setFirstName(firstName);
```

Comment:
setter in Domänenobjekten würde ich grundsätzlich probieren zu vermeiden, oder besser gesagt, nicht einfach generieren nur damit ich setter habe.

Reply:
Ok, mach sie private.